src/workbookManager.ts

```diff
@@ -78,6 +80,169 @@ export class WorkbookManager {
+            queryTableFields.removeChild(queryTableFields.lastChild);
+        }
+        var columnIndex = 1;
+        tableData.columnNames.forEach(columnName => {
+            const queryTableField = queryTableDoc.createElementNS(queryTableDoc.documentElement.namespaceURI, "queryTableField");
+            queryTableField.setAttribute("id", columnIndex.toString());
+            queryTableField.setAttribute("name", columnName);
+            queryTableField.setAttribute("tableColumnId", columnIndex.toString());
+            queryTableFields.appendChild(queryTableField);
+            columnIndex++;
+        });
+        queryTableFields.setAttribute("count", tableData.columnNames.length.toString());
+        queryTableDoc.getElementsByTagName("queryTableRefresh")[0].setAttribute("nextId", (tableData.columnNames.length + 1).toString());
+        return serializer.serializeToString(queryTableDoc);
+    }
+
+    private async updateSheetsInitialData(sheetsXmlString: string, tableData: TableData) {
+         const parser: DOMParser = new DOMParser();
+        const serializer = new XMLSerializer();
+        const sheetsDoc: Document = parser.parseFromString(sheetsXmlString, "text/xml");
+        const sheetData = sheetsDoc.getElementsByTagName("sheetData")[0];
+        while (sheetData.lastChild) {
+            sheetData.removeChild(sheetData.lastChild);
+        }
+        var rowIndex = 0;
+        const columnRow = sheetsDoc.createElementNS(sheetsDoc.documentElement.namespaceURI, "row");
+        columnRow.setAttribute("r", (rowIndex + 1).toString());
+        columnRow.setAttribute("spans", "1:" + (tableData.columnNames.length));
+        columnRow.setAttribute("x14ac:dyDescent", "0.3");
+        var colIndex = 0;
+        tableData.columnNames.forEach(column => {
+            const cell = sheetsDoc.createElementNS(sheetsDoc.documentElement.namespaceURI, "c");
+            cell.setAttribute("r", String.fromCharCode(colIndex + 65) + (rowIndex + 1).toString());
+            const cellData = sheetsDoc.createElementNS(sheetsDoc.documentElement.namespaceURI, "v");
+            cell.setAttribute("t", "str");
+            cellData.textContent = column;            
+            cell.appendChild(cellData);
+            columnRow.appendChild(cell);
+            colIndex++;
+        });
+        sheetData.appendChild(columnRow);
+        rowIndex++;
+        tableData.data.forEach((row) => {
+            const newRow = sheetsDoc.createElementNS(sheetsDoc.documentElement.namespaceURI, "row");
+            newRow.setAttribute("r", (rowIndex + 1).toString());
+            newRow.setAttribute("spans", "1:" + (row.length));
+            newRow.setAttribute("x14ac:dyDescent", "0.3");
+            var colIndex = 0;
+            row.forEach(cellContent => {
+                const cell = sheetsDoc.createElementNS(sheetsDoc.documentElement.namespaceURI, "c");
+                cell.setAttribute("r", String.fromCharCode(colIndex + 65) + (rowIndex + 1).toString());
```

Comment:
Instead of 65 use 'A' for clarity.

Also - consider this as a small util function - given a column and a row, get a reference.